pvlib/iotools/tmy.py

```diff
@@ -199,8 +204,15 @@ def read_tmy3(filename, coerce_year=None, recolumn=True):
     # unit must be in (D,h,m,s,ms,us,ns), but pandas>=0.24 allows unit='hour'
     data.index = data_ymd + pd.to_timedelta(shifted_hour, unit='h')
 
-    if recolumn:
+    if map_variables:
         data = _recolumn(data)  # rename to standard column names
+    elif recolumn:
+        if not map_variables:  # silence warning if map_variables is false
+            data = _recolumn(data)
+        elif map_variables is None:
+            data = _recolumn(data)
+            warnings.warn("recolumn parameter will be retired starting version 0.9.5,"
+                    "please use map_variables parameter instead.",DeprecationWarning)
```

Comment:
```suggestion
            warnings.warn("recolumn parameter will be retired starting version"
                          "0.9.5, please use map_variables parameter instead.",
                          DeprecationWarning)
```

Reply:
@ooprathamm I think the above suggestion should solve the stickler issue. You can click "commit suggestion" the suggestion with your existing contribution. Let me know anything is unclear.